pfcli/service/cloud.py

```diff
@@ -4,33 +4,57 @@
 
 from __future__ import annotations
 
+from abc import ABC, abstractmethod
 from dataclasses import dataclass
-from typing import Any, Callable, Dict, List, Optional, Tuple, Type, TypeVar, Union
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Generic,
+    List,
+    Optional,
+    Tuple,
+    Type,
+    TypeVar,
+    Union,
+)
 
 import boto3
 from azure.storage.blob import BlobServiceClient
-from botocore.client import BaseClient
 from botocore.exceptions import ClientError
+from mypy_boto3_s3.client import S3Client
```

Comment:
I encountered this error while running `pf deployment req-resp`.
Do you have to move the dependency from `dev` to `common` in `setup.py`?
```
    from mypy_boto3_s3.client import S3Client
ModuleNotFoundError: No module named 'mypy_boto3_s3'
```